client/ayon_shotgrid/addon.py

```diff
@@ -48,7 +48,7 @@ def get_local_storage_key(self):
     def create_shotgrid_session(self):
         from .lib import credentials
 
-        sg_username = os.getenv("AYON_SG_USERNAME")
+        sg_username = os.getenv("USER") or os.getenv("AYON_SG_USER")
```

Comment:
```suggestion
        sg_username = os.getenv("AYON_SG_USERNAME") or os.getenv("USER")
```

Reply:
It seems that the order should be flipped, right? Also is there any particular reason for proposing the original variable name?

Reply:
either way, in our studio there's never a case where one would point to something else but yeah I can revert it and keep the original name of the env.

I think I renamed it just to follow closer to the original env used in OP `OPENPYPE_SG_USER`

Reply:
you already merged it so I guess you will do the changes?

Reply:
yeah, sorry :D

Reply:
I think ideally we should be leveraging the existing login framework from Ayon... having multiple login dialogs is very cumbersome and there's really 0 need to have an authentication for SG as the whole plugin uses the API key to publish